packages/components/app/styles/components/tabs.scss

```
@@ -86,3 +85,23 @@ $indicator-height: 3px;
 .hds-tabs__tab-badge {
   margin-left: 6px;
 }
+
+.hds-tabs__tab-indicator {
+  position: absolute;
+  right: 0;
+  bottom: $indicator-height;
+  bottom: 0;
```

notice: there are two bottom declarations
```suggestion
  bottom: 0;
```

---

Sorry I missed that when I was refactoring. I will clean up